### ISSUE TITLE
Steam 64 ID not working

### ISSUE THREAD

**Issue by tugamars** [saul/demofile#98]
Hello, after the latest update to the demofile package, getting a players steamid64 isnt working as expected.

Code:
```javascript
console.log("Player: " + p.name + " - Steamid: " + p.steam64Id + " - " + p.steamId);
```

Output:
```
Player: SLiNKY - Steamid: 132148288220561409 - STEAM_1:1:15384085
Player: brA+ - Steamid: 37199231459524609 - STEAM_1:1:4330560
Player: rafafas7 <bru3 - Steamid: 471722210962178049 - STEAM_1:0:54915693
Player: Spi7 - Steamid: 340457341697654785 - STEAM_1:0:39634451
Player: Demonos - Steamid: 741665170418302977 - STEAM_1:1:86341189
```

Steam 32 works fine, but 64 is having problems.

**Comment by saul** [comment]
Hi @tugamars 

That's strange - I'll take a look when I get chance.

Thanks for the report.

**Comment by saul** [comment]
Fixed in v1.0.1
